docs/source/price_multiplier.rst

```diff
@@ -0,0 +1,11 @@
+Price Multiplier
+================
+
+.. toctree::
+   :maxdepth: 3
+
+   agents
```

Comment:
ok, this clearly comes from the old version, not sure why GitHub suggests that those are new and added in this PR...

Reply:
Weird. Yeah. I'm not sure.